cluster/src/main/java/io/scalecube/cluster/gossip/IGossipProtocol.java

```diff
@@ -12,10 +15,19 @@
  */
 public interface IGossipProtocol {
 
```

Comment:
It depends, if I generally want to have possibility to replace failure detection algorithm with something else - then I need start/stop methods on interface, if not then I don't need interface here at all... not decided here yet.